cpp/tests/test_save_results.cpp

```diff
@@ -112,6 +113,12 @@ TEST(TestSaveResult, save_result)
 
 TEST(TestSaveResult, save_result_with_params)
 {
+    //rng needs to be reseeded right before using parallel parameterstudies
+    //to keep the rest of the tests independent, we install a temporary RNG for this test
+    auto rng = mio::thread_local_rng();
```

Comment:
Do we now need to surround the example by the RNG? Please explain.

Reply:
ParameterStudy::run (the parallel version only) needs a freshly seeded RNG, otherwise selecting specific blocks of random numbers for each run doesn't work. For the tests, the RNG is only seeded once, so it's used up by the time this particular test is run. In the examples and the simulations, the RNG is seeded at startup, so it's fine. 

Anyway, this will change in the next iteration of the RNG, so this is temporary.